### DISCUSSION TITLE
What's the actual default theme name?

### DISCUSSION THREAD

**Discussion by brielov** [helix-editor/helix#8575] [Q&A] [answered]
Is there an actual name for the default purple theme? I'm trying to find a matching theme for alacritty but I don't really know how to look for it.

**Comment by gabydd** [comment, marked as answer]
its called colibri: https://github.com/archseer/colibri.vim you probably have to make the theme yourself though, not sure if @archseer has a alacritty theme for it

**Reply by nshern** [reply]
You can extract the colors for alacritty from @archseer s [dotfiles](https://github.com/archseer/dotfiles/blob/master/config/alacritty/alacritty.yml)

**Reply by brielov** [reply]
That worked like a charm! Thanks 🦾
